frontend/src/app/shared/components/autocompleter/project-autocompleter/project-autocompleter.component.ts

```diff
@@ -112,6 +117,8 @@ export class ProjectAutocompleterComponent implements ControlValueAccessor {
 
   @Input() public appendTo = '';
 
+  @Input() public isInlineContext = false;
```

Comment:
what is the advantage of using an input in combination with a hostbinding, instead of simply using `ngClass` in whatever template this component is used? IMHO it kinda hides the effect, when simply looking on the parent template.

Reply:
In my eyes it has two main advantages:

1. Using an input is kind of a documentation the inside the belonging component. If you'd only use a class in the parent component, you'd not see its effect in the code. You'd have to go to the component style sheet to check it which normally nobody does, tbh. 
2. In general using an `ngClass` is very error-prone to spelling mistakes. As soon as someone uses the wrong class, it won't work and nobody would notice because there is no compile error.

Reply:
agreed, approved it is then